integrated_channels/sap_success_factors/transmitters/courses.py

```diff
@@ -3,28 +3,47 @@
 """
 from __future__ import absolute_import, unicode_literals
 from integrated_channels.sap_success_factors.transmitters import SuccessFactorsTransmitterBase
+from integrated_channels.sap_success_factors.models import CatalogTransmissionAudit
+from requests import RequestException
 
 
 class SuccessFactorsCourseTransmitter(SuccessFactorsTransmitterBase):
     """
     This endpoint is intended to carry out an export of course data to SuccessFactors for a given Enterprise.
-
-    Implementation will look something like:
-
-    Perform basic validation on the input (e.g. is this Enterprise configured for SuccessFactors?)
-
-    Retrieve oauth access token from SuccessFactors, based on OCNWebServicesEnterpriseCustomerConfiguration.
-    This can either initiate generating a new token, or using an unexpired token which we have cached.
-
-    Find which catalog is associated with the enterprise customer.
-
-    Retrieve course data in the catalog from the Catalog API.
-
-    Transform course data into the format expected by SuccessFactors.
-
-    Send the transformed course data to SuccessFactors using the configured endpoint
-    (taken from OCNWebServicesConfiguration and OCNWebServicesEnterpriseCustomerConfiguration) and the oauth token.
-
-    Record information about the success/failure of posting the course data to SuccessFactors in
-     CourseContentExportAudit.
     """
+
+    def __init__(self, enterprise_configuration):
```

Comment:
Note that if you're not doing anything with the data passed into `__init__`, you can just not implement it altogether and let it fall through to the base class.